<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Node</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>35</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueLE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>65</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>65</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tag:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>35</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Text/Combo Box</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Radio Button/Check Box</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="levelCB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>5</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>150</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>95</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputTE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>95</y>
     <width>260</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
